app/components/action-button.js

```diff
@@ -23,18 +41,22 @@ const ActionButton = props => {
           href={action}
           target={action[0] === '/' ? '_self' : '_blank'}
           {...otherProps}
-        ></a>
+        >
+          {renderButtonWithIcon()}
```

Comment:
I don't think this will render correctly for the "string" case.  Why don't you make a story case for "relative path with icon" and make adjustments if necessary.   When an action is the path, we are rendering an anchor that looks like a button, rather than a button, which makes things more apparent for web crawling.